crates/torii/client/src/client/storage.rs

```diff
@@ -86,28 +85,6 @@ impl ModelStorage {
         raw_keys: Vec<FieldElement>,
         raw_values: Vec<FieldElement>,
     ) -> Result<(), Error> {
-        let model_name =
```

Comment:
We could potentially check that, but only if the layout if `Fixed`, which means everything is packed as before, hence the values should match.

We can address specific tests for that in an other PR, since related to some changes to the namespace will need test too.